decision-making/voting-on-proposals.md

```diff
@@ -0,0 +1,30 @@
+# Decision-making
+
+### Proposal submission
+Proposal is submitted by opening pull request to this repository.
+Initiator must present the idea on the architecture meeting.
+The announcement should include problem description the proposal is trying to resolve as well as solution(s).
+
+### Discussion
+There must be minimum 2 weeks of discussion in the pull request thread.
+After that initiator may decide when to start the vote.
+
+### Voting
+All eligible voters must get notification about the voting start.
+A valid voting period must be announced when voting is started, must be greater that 2 weeks and must not be changed during the vote.
+
+### Approval
+The number of `approve` votes must be greater than or equal to the number of `reject` votes multiplied by two in order to merge pull request.
+In case of multi-choice voting (implementation details) the decision may be taken by simple plurality. This means that the voting option with the most votes wins. If there are multiple options with the most number of votes, the initiator can choose one of them.
+Once pull request is merged the proposal is accepted.
+
+# Eligible voters
+The group of eligible voters should be established. The group should represent the Magento community and include internal developers and architects (40%), Magento maintainers (30%), extension developers and partner contributors (30%).
```

Comment:
Is there a requirement for how many votes are needed or is the number of votes required determined by the number of eligible people in the voter pool? Also, is there a requirement for percentage of votes from a particular category?

Reply:
Will add this info as well. This is the rough draft I wrote to understand if we want to move in this direction. It definitely requires defining all rules and use cases.

Reply:
We might want to have some kind of rotation in "eligible voters" groups, otherwise there is a risk of forming a group biased to specific questions

Reply:
I would not introduce tight coupling to the maintainers group establishing some proportion, because sometimes we have contributors who are not maintainers but actively participating in Magento Development process, there are other influencers who do not maintain Magento either.

This supposed to be a new role - Community Architect.

Magento team should be also represented widely, for example, there should be people from Cloud and MOM.

The proportion may vary, thus I would like not to mention it here at all, and we may extend the group of eligible voters with both internal and external participants. We should be flexible enough here.

Reply:
Is there a minimum threshold of cast votes from the voters pool? so at least 50% of people eligible to vote need to vote otherwise the vote will... fail? need to be done again? will a failed vote need to be rewritten? or can it be voted on again immediately?

Reply:
aka, what is quorum?